src/ResourceManager/Compute/Commands.Compute/Models/PSVirtualMachine.cs

```diff
@@ -33,6 +33,7 @@ public string ResourceGroupName
         {
             get
             {
+                if (string.IsNullOrEmpty(Id)) return null;
```

Comment:
This is a generated model class code, and it shall not be changed manually. It's better to update the script.